### PR TITLE
Added Inital UI Changes for v2 workflow representation screens.

### PR DESCRIPTION
**Pull request by Jonsy13**
Signed-off-by: Jonsy13 <vedant.shrotria@mayadata.io>

## Proposed changes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

**This PR will add UI changes for v2 workflow representation and logs screens.**

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [X] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [X] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [X] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [X] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

## Special notes for your reviewer:
**There are still changes required with respect to redux and colors. They will come in upcoming PRs.**

Screenshot from 2021-03-03 13-08-08
Screenshot from 2021-03-03 13-13-41
Screenshot from 2021-03-03 13-08-04
Screenshot from 2021-03-03 13-08-13
Screenshot from 2021-03-03 13-08-16
Screenshot from 2021-03-03 13-08-26
Screenshot from 2021-03-03 13-14-44

### CODE REVIEW

**Review by amityt — APPROVED** [review]

File: litmus-portal/frontend/src/views/WorkflowDetails/NodeLogs/index.tsx [hunk 1]
```diff
@@ -151,13 +145,11 @@ const NodeLogs: React.FC<NodeLogsProps> = ({
...
-    <Unimodal
-      open={logsOpen}
-      handleClose={handleClose}
-      hasCloseBtn
-      textAlign="left"
-    >
-      <div className={classes.root}>
+    <div className={classes.root}>
+      <Typography className={classes.logsHeading}>
+        Logs for this step
```
Comment (S-ayanide) [inline litmus-portal/frontend/src/views/WorkflowDetails/NodeLogs/index.tsx:150]: Can we add this to translation as well!

File: litmus-portal/frontend/src/views/WorkflowDetails/NodeLogs/styles.ts [hunk 2]
```diff
@@ -3,11 +3,27 @@ import { makeStyles, Theme } from '@material-ui/core/styles';
...
-    background: theme.palette.common.black,
-    color: theme.palette.text.secondary,
+    [theme.breakpoints.up('lg')]: {
+      height: '15rem',
+    },
+    margin: theme.spacing(2, 0, 0, 2),
+    height: '100%',
+    background: theme.palette.cards.header,
+    color: theme.palette.text.primary,
+    textAlign: 'left',
```
Comment (S-ayanide) [inline litmus-portal/frontend/src/views/WorkflowDetails/NodeLogs/styles.ts:26]: Should be aligned to the left by default

File: litmus-portal/frontend/src/views/WorkflowDetails/NodeLogs/styles.ts [hunk 3]
```diff
@@ -3,11 +3,27 @@ import { makeStyles, Theme } from '@material-ui/core/styles';
...
+  logs: {
+    padding: theme.spacing(1.5),
     overflowY: 'scroll',
-    background: theme.palette.common.black,
-    color: theme.palette.text.secondary,
+    [theme.breakpoints.up('lg')]: {
+      height: '15rem',
+    },
+    margin: theme.spacing(2, 0, 0, 2),
+    height: '100%',
```
Comment (S-ayanide) [inline litmus-portal/frontend/src/views/WorkflowDetails/NodeLogs/styles.ts:23]: Height is set to 100% by default

File: litmus-portal/frontend/src/views/WorkflowDetails/WorkflowInfo/index.tsx [hunk 4]
```diff
@@ -1,175 +1,103 @@
...
-    setData({
-      ...data,
-      currentRunningNodes,
-      executedNodes,
-    });
-  }, [execution_data.nodes, execution_data.phase]);
 
   return (
-    <div className={classes.root}>
+    <div className={tab === 1 ? classes.rootLight : classes.rootHeader}>
```
Comment (S-ayanide) [inline litmus-portal/frontend/src/views/WorkflowDetails/WorkflowInfo/index.tsx:23]:
```suggestion
    <div className={`${classes.root} ${tab === 1 ? classes.light : classes.header}`}>
```

File: litmus-portal/frontend/src/views/WorkflowDetails/WorkflowInfo/index.tsx [hunk 5]
```diff
@@ -1,175 +1,103 @@
...
-          <Typography>
-            <span className={classes.bold}>
-              {t('workflowDetailsView.workflowInfo.param.state')}:
-            </span>{' '}
-            {execution_data.phase}
+      <Typography className={classes.descTextBold}>
+        {t('workflowDetailsView.workflowInfo.header')}
+      </Typography>
+      <div className={classes.headerFlex}>
+        <div className={classes.headerItemFlex}>
```
Comment (S-ayanide) [inline litmus-portal/frontend/src/views/WorkflowDetails/WorkflowInfo/index.tsx:29]: Can we structure the new code better! As in 

class header
-- text 

Something like this. Reference: `ChooseAWorkflowAgent` file

File: litmus-portal/frontend/src/views/WorkflowDetails/WorkflowInfo/index.tsx [hunk 6]
```diff
@@ -1,175 +1,103 @@
...
-          <Typography>
-            <span className={classes.bold}>
-              {t('workflowDetailsView.workflowInfo.param.startTime')}:
-            </span>{' '}
-            {timeDifference(execution_data.startedAt)}
+          <Typography className={classes.resilliencyScore}>100%</Typography>
+        </div>
+        <div className={classes.headerItemFlex}>
+          <Typography className={classes.textBold}>
+            {t('workflowDetailsView.workflowInfo.runTime.runTimeHeader')}
```
Comment (S-ayanide) [inline litmus-portal/frontend/src/views/WorkflowDetails/WorkflowInfo/index.tsx:37]: Can we have the restructuring for all the next files!

**Review by S-ayanide — CHANGES_REQUESTED** [review]

**Comment by S-ayanide** [comment]
Can you please resolve the DeepScan issues!

**Review by S-ayanide — APPROVED** [review]

**Review by amityt — APPROVED** [review]

**Pull request merged**